packages/aws-rfdk/lib/deadline/lib/worker-fleet.ts

```diff
@@ -279,10 +279,21 @@ abstract class WorkerInstanceFleetBase extends Construct implements IWorkerFleet
  *
  * Resources Deployed
  * ------------------------
- * 1) An AutoScalingGroup to maintain the number of instances;
- * 2) An Instance Role and corresponding IAM Policy;
- * 3) A script asset which is uploaded to your deployment bucket used to configure the worker so it can connect to the Render Queue
- * 4) An aws-rfdk.CloudWatchAgent to configure sending logs to cloudwatch.
+ * - An EC2 Auto Scaling Group to maintain the number of instances.
+ * - An Instance Role and corresponding IAM Policy.
+ * - An Amazon CloudWatch log group that contains the Deadline Worker, Deadline Launcher, and instance-startup logs for the instances
+ *   in the fleet.
+ *
+ * Security Considerations
+ * ------------------------
+ * - The instances deployed by this construct download and run scripts from your CDK bootstrap bucket when that instance
```

Comment:
We currently warn the user if they do not encrypt the worker fleets EBS volumes should we have notes about that here as well?

Reply:
Yeah, that would make sense.